caramel/scripts/generate_ca.py

```diff
@@ -197,15 +198,10 @@ def main():
     env = bootstrap(args.inifile)
     settings, closer = env["registry"].settings, env["closer"]
 
-    ca_cert = settings.get("ca.cert")
-    ca_key = settings.get("ca.key")
-    if not ca_cert:
-        print("Missing 'ca.cert' in ini file")
-        closer()
-        exit()
-
-    if not ca_key:
-        print("Missing 'ca.cert' in ini file")
+    try:
+        ca_cert, ca_key = config.get_ca_cert_key_path(args, settings)
```

Comment:
The name "ca_cert"  doesn't match the content. adding _path would help. Especially for code where one could expect "ca_key" and "ca_cert" to actually contain certificates.

Just because the old code was dubious, does not mean new code cannot be better :)